tests/cases/.gitignore

```diff
@@ -0,0 +1 @@
+*.out
```

Comment:
don't ignore the `.out` file. It should be tracked and we need to guarantee no `.out` will be added.

Reply:
Should it be guaranteed that no `.out` is added if we ignore it?

Reply:
I see. We can find the test failure by checking the existence of these `.out` files.